core/templates/pages/contributor-dashboard-page/services/contribution-opportunities-backend-api.service.ts

```diff
@@ -166,17 +167,22 @@ export class ContributionOpportunitiesBackendApiService {
   }
 
   async fetchReviewableTranslationOpportunitiesAsync(
-      topicName: string
+      topicName: string,
+      languageCode?: string
   ): Promise<FetchedReviewableTranslationOpportunitiesResponse> {
     const params: {
       topic_name?: string;
+      language_code?: string;
     } = {};
-    if (topicName !== constants.TOPIC_SENTINEL_NAME_ALL) {
+    if (topicName && topicName !== constants.TOPIC_SENTINEL_NAME_ALL) {
```

Comment:
Is this checking for null or empty string?

Reply:
Empty. Fixed!